Source/WebCore/editing/BreakBlockquoteCommand.cpp

```diff
@@ -128,17 +128,17 @@ void BreakBlockquoteCommand::doApply()
     ASSERT(startNode);
     // Split at pos if in the middle of a text node.
     if (is<Text>(*startNode)) {
-        Text& textNode = downcast<Text>(*startNode);
-        if ((unsigned)pos.deprecatedEditingOffset() >= textNode.length()) {
-            if (auto* nextNode = NodeTraversal::next(*startNode))
-                startNode = nextNode;
+        Ref textNode = downcast<Text>(*startNode);
+        if ((unsigned)pos.deprecatedEditingOffset() >= textNode->length()) {
```

Comment:
We should use a C++ cast for unsigned. Not sure if it's just a `static_cast<unsigned>` in this case.